src/lib/user-ip-address.test.js

```diff
@@ -0,0 +1,24 @@
+const userIpAddress = require("./user-ip-address");
+
+describe("headers", () => {
+  let req;
+
+  beforeEach(() => {
+    const setup = setupDefaultMocks();
+    req = setup.req;
+  });
+
+  it("should return null", () => {
```

Comment:
The header value we're being given is of the format `for=123.456.789.012;host=subdomain.example.gov.uk;proto=https`

I'd like to see tests for this exact string, and one without the for= block, and for an empty string as well.

Reply:
add the test case as well

Reply:
In my opinion - we should also add a ipv6 test to test the usecase - `net.isIPv6(clientIp)` but that's up for a convo